enclave/cert.c

```diff
@@ -634,6 +635,55 @@ oe_result_t oe_cert_read_pem(
     return result;
 }
 
+oe_result_t oe_cert_read_der(
+    oe_cert_t* cert,
+    const void* der_data,
+    size_t der_size)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    Cert* impl = (Cert*)cert;
+    mbedtls_x509_crt* crt = NULL;
+    int rc = 0;
+
+    /* Clear the implementation */
+    if (impl)
+        memset(impl, 0, sizeof(Cert));
+
+    /* Check parameters */
+    if (!der_data || !der_size || !cert)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    /* Allocate memory for the certificate */
+    if (!(crt = mbedtls_calloc(1, sizeof(mbedtls_x509_crt))))
```

Comment:
Personally I think it's bad for a core OE file to directly hard code mbedTLS specific API calls instead of going through an abstraction layer.   I thought that was being addressed.

Reply:
This cert.c is part of the crypto implementation for the enclave, which depends solely on the mbedtls library. 
Before that dependency was replaced/removed, we will have to keep it this way.

Reply:
I can't parse the last sentence in your reply.   Do you mean "Until that is done"?  meaning it's future work to fix this?  Or what?   I still maintain its bad to call mbedtls APIs directly from OE core code.